### ISSUE TITLE
Allow exporting all parameters to a .param file

### ISSUE THREAD

**Issue by Williangalvani** [ArduPilot/UAVLogViewer#255]
We already have all the parameters. All we have to do is generate a file from the parameters widget.

Bonus points: Generate a file with only the ones currently showing (with the filter applied)

**Comment by palm369** [comment]
Yes please! Or at least make them highlightable so one can copy past the list into a file. Right now I cant find any way to get the params out a log without MP.
